regression/goto-instrument/restrict-function-pointer-by-name-local/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.c
+--restrict-function-pointer-by-name main::1::fp/f
+\[main\.assertion\.1\] line \d+ dereferenced function pointer at main\.function_pointer_call\.1 must be f: SUCCESS
+\[main\.assertion\.2\] line \d+ assertion fp\(\) == 1: SUCCESS
+^EXIT=0$
+^SIGNAL=0$
+--
+--
```

Comment:
:no_entry_sign: please make sure _all_ tests have an explanation of what they are supposed to be testing. The name is necessarily terse, and in 6 months we definitely will get confused.

Reply:
Added